### DISCUSSION TITLE
Galaxy badge

### DISCUSSION THREAD

**Discussion by severdia** [Schweinepriester/achievement-testarea#29] [Q&A]
What is needed?

**Comment by xommmax** [comment]


**Reply by severdia** [reply]
You got it!

**Reply by xommmax** [reply]
@severdia mark as answer please 😆🥰

**Reply by severdia** [reply]
This is the answer!

**Reply by xommmax** [reply]
great, and a second one)

**Reply by severdia** [reply]
Sorry, I had to move it to Q&A to see the button. Fixed!

**Reply by xommmax** [reply]
oh, that is why id didnt work

**Comment by severdia** [comment]
https://github.com/Schweinepriester/achievement-testarea/discussions/30

**Comment by hoseinpur** [comment]
I want to test this too :(

**Comment by pulkitxm** [comment]
star my  [repo](https://github.com/Pulkitxm/Delta-Apna-College), I'll do it for yours
 [https://github.com/Pulkitxm/Delta-Apna-College](https://github.com/Pulkitxm/Delta-Apna-College)
Have a nice Day !

**Comment by pulkitxm** [comment]
best Answer
